apps/new-tab/src/App.tsx

```
@@ -1,1702 +1,737 @@
+type Widget =
+  | "clock"
+  | "date"
+  | "stopwatch"
+  | "todo"
+  | "bookmarks"
+  | "nature"
+  | "pomodoro"
+  | "focus"
+  | "ambience";
+
+type Bookmark = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Bookmark'. Consider to delete it or rename it.

'Bookmark' is defined here:


(lint/suspicious/noRedeclare)